spec/index.bs

```diff
@@ -6558,23 +6558,25 @@ pipeline, and have the following members:
                 [=Device timeline=] |initialization steps|:
 
                 1. Let |limits| be |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.
+                1. Let |bindGroupLayouts| be a copy of |descriptor|.{{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
+                1. For each |bindGroupLayout| in |bindGroupLayouts|:
```

Comment:
We need to reify the empty bind group layouts to null as well.

Reply:
Done.

Reply:
I was going to comment we needed to do this for the "auto" layout algorithm too, but I realized that this fixes that implicitly!